spec/features/analysis/user_can_visit_the_dashboard_to_see_analytics_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+describe "User visits the dashboard and sees analytics" do
+  scenario "user sees a count of jobs by level of interest" do
+    visit "/dashboard"
```

Comment:
leverage your path helpers! and make sure you create them if you're creating custom routes :thumbsup: